content/blog/2021-04-30-april-21-community-gems.md

```diff
@@ -0,0 +1,183 @@
+  ideas for running CML pipeline reports and more.
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC community. 
+  This month: remote storage integration, removing old experiments,
+  ideas for running CML pipeline reports  and more.
+picture: 2021-04-30/gems-cover.png
+author: jeny_defigueiredo
+commentsUrl: https://discuss.dvc.org/t/april-21-community-gems/
+tags:
+  - Discord
+  - Gems
+  - DVC
+  - CML
+  - Remote Storage
+---
+
+Each month we go through our Discord messages to pull out some of the best
+questions from our community. AKA: Community Gems. 💎 This month we'd like to
+thank asraniel, Julian, mattlbeck, Ahti, **\_**, yikeqicn, lexzen, EdAb,
+FreshLettuce for inspiring this month's gems!
+
+As always, [join us in Discord](https://discord.com/invite/dvwXA2N) to get all
+your DVC and CML questions answered!
+
+## DVC
+
+### [Can directories and subdirectories be created on OwnCloud with a WebDav remote when I run dvc push if they don't already exist?](https://discord.com/channels/485586884165107732/485596304961962003/831472645508694046)
+
+This likely means that there is an issue with either the credentials or
+permissions set up in WebDav. Make sure that you are using the correct protocol
+and provide a username and/or token or password.
+
+If you use webdavs for your URL and use the `ask_password` parameter for your
+config, this should clear up the issue.
+
+### [What procedures would I use if I needed to remove unwanted or outdated experiments and corresponding data in the cache?](https://discord.com/channels/485586884165107732/485596304961962003/831127462544146482)
+
+You can use `dvc exp gc` with corresponding flags to delete the experiments
+themselves and there are a number of other flags that can be used with
+`dvc exp gc` depending on your needs.
+
+Alternatively, you can remove branches that are no longer required using
+`dvc exp remove`, then run `dvc gc --all-branches --all-tags`. DVC will then go
+through each branch and tag revisions; will keep them, and will purge the rest.
+
+Note that `--all commits` is the safest flag to use, though it will not remove
+much of the cache. Finally, when you delete what's in cache, it does not delete
+the directory completely. It leaves the directory empty in your repo. To find
+info and comment on this known issue, head to
+[Issue #3375 here.](https://github.com/iterative/dvc/issues/3375) Your input is
+ALWAYS valuable to us! 🙏🏼
+
+### [If I have two cloud folder links added to the DVC config, I'm able to push the data to the default one. How could I push the data to the other cloud folder?](https://discord.com/channels/485586884165107732/563406153334128681/833176227762274364)
+
+You're looking for the `-r / --remote` option for `dvc push`. The command looks
+like this:
+
+```
+dvc push --remote name_of_remote_storage
+```
+
+This will upload tracked files or directories to remote storage based on the
```

Comment:
let's skip `based on the current _dvc.yaml_ and _.dvc_ files.` - it's not important here I think. I would clarify that it'll push specifically to the < name_of_remote_storage >